engine/apps/api/serializers/alert_receive_channel.py

```diff
@@ -343,7 +324,6 @@ def _handle_core_template_updates(self, data, ret):
 
     def to_representation(self, obj):
         ret = super().to_representation(obj)
-        ret = self._get_templates_to_show(ret)
```

Comment:
do we need this function anymore?

https://github.com/grafana/oncall/pull/2064/files#diff-b8990ca4b389382d4e7b6f58fd7c1bf0773c85d3a86db2a4a4486e2459dcf8ffR337